web/yo/app/scripts/app.js

```diff
@@ -218,9 +198,17 @@ OncoKB.curateInfo = {
         },
         nccn: {
             type: 'NCCN'
+        }
+    },
+    Diagnostic: {
+        level: {
+            type: 'string'
         },
-        interactAlts: {
-            type: 'InteractAlts'
+        description: {
+            type: 'string'
+        },
+        short: {
```

Comment:
I'm not really sure about this prop, this is for additional info? We were using short for other purposes and later on had to use short for additional info. Should we keep the consistency or use the proper language?